tests/check_ipc.c

```diff
@@ -122,26 +122,19 @@ exit_handler(int32_t rsignal, void *data)
 static void
 set_ipc_name(const char *prefix)
 {
-	/* We have to make the server name as unique as possible given
-	 * the build- (seconds part of preprocessor's timestamp) and
-	 * run-time (pid + lower 16 bits of the current timestamp)
-	 * circumstances, because some build systems attempt to generate
-	 * packages for libqb in parallel.  These unit tests are run
-	 * during the package build process.  2+ builds executing on
-	 * the same machine (whether containerized or not because of
-	 * abstract unix sockets namespace sharing) can stomp on each
-	 * other's unit tests if the ipc server names aren't unique... */
-
-	/* single-shot grab of seconds part of preprocessor's timestamp */
-	static char t_sec[3] = "";
-	if (t_sec[0] == '\0') {
-		const char *const found = strrchr(__TIME__, ':');
-		strncpy(t_sec, found ? found + 1 : "-", sizeof(t_sec) - 1);
-		t_sec[sizeof(t_sec) - 1] = '\0';
+	FILE *f;
+	char process_name[256];
+
+	/* The process-unique part of the IPC name has already been decided
+	 * and stored in the file ipc-test-name.
+	 */
```

Comment:
It may be nice to have some fallback when `ipc-test-name` file does not exists because right now, result is going to be random bytes from memory.